README.md

```diff
@@ -1,69 +1,32 @@
 ## Configure Development Environment
 
 ### Install dev/build tools
-- Node.js 16.x or higher
 - [Yarn](https://yarnpkg.com)
+- Node.js 16.x or higher (installed with yarn)
```

Comment:
I'm confused by "installed with Yarn". Both install paths I'm seeing for Yarn require having Node installed first.

[Classic instructions](https://classic.yarnpkg.com/en/docs/install#windows-stable) are to run `npm install --global yarn`. Quoting that page:

> It is recommended to install Yarn through the npm package manager, which comes bundled with Node.js when you install it on your system.

[New instructions](https://yarnpkg.com/getting-started/install) use corepack: `corepack enable; corepack prepare yarn@stable --activate`. Corepack also requires Node to be installed first:

> Corepack is included by default with all Node.js installs

Is there a documented/preferred way of installing Yarn first and then using Yarn to install Node.js?